Documentation/Changes.txt

```diff
@@ -15,11 +15,14 @@ Version 1.0.6
 * Fix TR4 SKELETON spawn when used with OCB 3
 * Fix TR4 SAS teleporting over the blocks he walks by.
 * Fix enemy projectile effect colours.
+* Fix switches with ocb 3, 5 and 6 that failed to work in some circunstances.
```

Comment:
"OCB" should be capitalized, and "circumstances", not "circunstances".

Reply:
Done! :D